react-native-is-edge-to-edge/README.md

```diff
@@ -0,0 +1,13 @@
+# react-native-is-edge-to-edge
+
+A companion for libraries authors.
+
+## API
```

Comment:
Probably already on your todo list, but a install instruction and maybe a mention in the main README.md file might be a good idea

Reply:
Yes, it's on my list!